app/main/views/withdraw_brief.py

```diff
@@ -26,7 +48,11 @@ def withdraw_a_brief(framework_slug, lot_slug, brief_id):
     if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id, allowed_statuses=['live']):
         abort(404)
 
-    data_api_client.withdraw_brief(brief_id, current_user.email_address)
-    flash(BRIEF_WITHDRAWN_MESSAGE.format(brief=brief), "success")
+    if "withdraw_confirmed" in request.form:
+        data_api_client.withdraw_brief(brief_id, current_user.email_address)
+        flash(BRIEF_WITHDRAWN_MESSAGE.format(brief=brief), "success")
+        return redirect(url_for(".buyer_dos_requirements"))
```

Comment:
I don't think we need "withdraw_confirmed" here or in the button any more, since Cancel is a link